pkg/terminal/command.go

```diff
@@ -917,11 +918,58 @@ func (t *Term) formatGoroutine(g *api.Goroutine, fgl formatGoroutineLoc) string
 		locname = "Start"
 		loc = g.StartLoc
 	}
-	thread := ""
+
+	buf := new(bytes.Buffer)
```

Comment:
We should use `strings.Builder` here. It's slightly more efficient in the `String` method, and shows intent a little better.

https://golang.org/pkg/strings/#Builder